### DISCUSSION TITLE
ES Modules support

### DISCUSSION THREAD

**Discussion by ABSanthosh** [deta/cloud-docs#286] [Help]
What's the status of ES Modules?
From [this](https://discord.com/channels/827546555200438332/831945258228908112/940524295300448257) conversation in discord,
I'm assuming it will be supported soon.

Thanks!

**Comment by KartikSoneji** [comment]
Hi, any updates on this?
Additionally, is this part open-sourced so I can contribute to it?

**Reply by jnsougata** [reply]
Sadly no!
